docs/operator-manual/custom-styles.md

```diff
@@ -96,4 +96,25 @@ Note that the CSS file should be mounted within a subdirectory of the "/shared/a
 ## Developing Style Overlays
 The styles specified in the injected CSS file should be specific to components and classes defined in [argo-ui](https://github.com/argoproj/argo-ui).
 It is recommended to test out the styles you wish to apply first by making use of your browser's built-in developer tools.  For a more full-featured
-experience, you may wish to build a separate project using the [Argo CD UI dev server](https://webpack.js.org/configuration/dev-server/).
\ No newline at end of file
+experience, you may wish to build a separate project using the [Argo CD UI dev server](https://webpack.js.org/configuration/dev-server/).
+
+## Banners
+
+> v2.0
```

Comment:
Please remove this version indicator. We don't use it anymore since we have versioned documentation (gotta clean up all the remainders as well).

Reply:
Updated. I saw them on different pages as well. Will try to clean them in a separate chore PR.